about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Secular+One&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/ab2d2d26a1.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./css/index.css" />
+    <title>DeveloperWeek - Conference & Expo, SF Bay Area</title>
+    <script src="./scripts/menu.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <nav class="mobile-menu">
```

Comment:
- [ ] [OPTIONAL] When the aside menu is open, users can still scroll the body section of the website. To avoid this situation, you can easily set the `body{ overflow: hidden }` when users click on hamburger button, and when users close the aside `menu: body{ overflow: auto; `}  👍🏻